lms/djangoapps/grades/tests/test_new.py

```diff
@@ -279,6 +281,25 @@ def test_subsection_grade_feature_gating(self, feature_flag, course_setting):
         self.assertEqual(mock_read_saved_grade.called, feature_flag and course_setting)
 
 
+class ZeroGradeTest(GradeTestBase):
```

Comment:
Thanks @efischer19.

I was also concerned about adding a test with rendering the existing progress page with a `ZeroGrade`.  I'd like to make sure that all the APIs/data the progress page requires is available for `ZeroGrade` objects.